sha2/Cargo.toml

```diff
@@ -22,6 +22,10 @@ cfg-if = "1.0"
 cpufeatures = "0.2"
 sha2-asm = { version = "0.6.1", optional = true }
 
+[target.'cfg(all(target_os = "zkvm", target_arch = "riscv32"))'.dependencies]
+risc0-zkvm-platform = "0.12"
```

Comment:
I wonder if there's a way to late-bind this dependency. Otherwise seems like we need to keep bumping this.

Reply:
Alternatively, maybe leave off the dependency and include the extern "C" definition for sys_sha_buffer in risc0.rs?

Reply:
When I was working on this before I did not find any nice way to do late-binding, either for extern Rust or C dependencies with Cargo. Do y'all have any pointers that might be helpful in that regard?